pants

```diff
@@ -45,12 +43,28 @@ function tempdir {
   mktemp -d "$1"/pants.XXXXXX
 }
 
+function get_pants_ini_config_value {
+  config_key="$1"
+  valid_delimiters="[:=]"
+  optional_space="[[:space:]]*"
+  prefix="^${config_key}${optional_space}${valid_delimiters}${optional_space}"
```

Comment:
It looks like the old regex used to allow a leading optional space before the key... which is odd? It would seem to make more sense the way you have it here.